python/bufr/encoders/zarr/encoder.py

```diff
@@ -0,0 +1,159 @@
+
+import os
+import re
+from typing import Union
+
+import zarr
```

Comment:
Is "zarr" off-the-shelf/standard Python package?

Reply:
It must be installed separatly (pip install zarr). The python package is the official implementation for that format.